lib/db_utils.go

```diff
@@ -202,10 +202,29 @@ var (
 	_PrefixRecloutedPostHashReclouterPubKeyRecloutPostHash = []byte{46}
 	_PrefixDiamondedPostHashDiamonderPKIDDiamondLevel      = []byte{47}
 
+	// Prefixes for NFT ownership entries:
+	// 	<prefix, NFTPostHash [32]byte, SerialNumber uint64> -> NFTEntry
+	_PrefixPostHashSerialNumberToNFTEntry = []byte{48}
+	//  <prefix, PublicKey [33]byte, IsForSale bool, BidAmountNanos uint64, NFTPostHash[32]byte, SerialNumber uint64> -> NFTEntry
+	_PrefixPublicKeyIsForSaleBidAmountNanosPostHashSerialNumberToNFTEntry = []byte{49}
+
+	// Prefixes for NFT bids:
+	_PrefixPostHashSerialNumberBidNanosBidderPKID   = []byte{50}
+	_PrefixBidderPKIDPostHashSerialNumberToBidNanos = []byte{51}
+
+	// <prefix, PublicKey [33]byte> -> uint64
+	_PrefixPublicKeyToBitCloutBalanceNanos = []byte{52}
+	// Block reward prefix (used for deducting immature block rewards from bitclout balances):
+	// <hash BlockHash> -> <pubKey [33]byte, uint64 blockRewardNanos>
+	_PrefixPublicKeyBlockHashToBlockReward = []byte{53}
```

Comment:
I like to elaborate a bit more when we need to add things like this. E.g. here I would add:

This index is needed because block rewards take N blocks to mature, which means we need
a way to deduct them from balance calculations until that point. Without this index, it would be
impossible to figure out which of a user's UTXOs have yet to mature.